package.json

```diff
@@ -21,6 +21,7 @@
     "babel-eslint": "^5.0.0",
     "babel-plugin-check-es2015-constants": "^6.5.0",
     "babel-plugin-syntax-trailing-function-commas": "^6.5.0",
+    "babel-plugin-syntax-object-rest-spread": "^6.3.13",
```

Comment:
We need `babel-plugin-syntax-flow` as well since we explicitly use it (at least for npm2 - npm3 will likely make it available at the same level when the transform is installed). That's why travis is failing to do anything right now.